samples/samples/test_autocommit.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+import uuid
+
+from google.api_core.exceptions import DeadlineExceeded
+from google.cloud import spanner
+import pytest
+
+import autocommit
```

Comment:
```suggestion
from . import autocommit
```

AFAIK it's never a good idea to use old-style relative imports, but maybe this is the recommended style for standalone docs?

Note that you'll need an `__init__.py` file in this package for this suggestion to work.